src/librustc_session/session.rs

```diff
@@ -548,7 +548,7 @@ impl Session {
         self.opts.debugging_opts.asm_comments
     }
     pub fn verify_llvm_ir(&self) -> bool {
-        self.opts.debugging_opts.verify_llvm_ir || cfg!(always_verify_llvm_ir)
+        self.opts.debugging_opts.verify_llvm_ir || option_env!("RUSTC_VERIFY_LLVM_IR").is_some()
```

Comment:
The `cfg` was set in `librustc_middle/build.rs`.
Looks like nobody noticed that it stopped working.